### DISCUSSION TITLE
It's too slow to drag the slider in the container.Split

### DISCUSSION THREAD

**Discussion by fayfive** [fyne-io/fyne#3241] [Q&A] [answered]
It's too slow to drag the slider in the `container.Split`
Here is my code. I combine `split` and `scroll` and It's too slow to drag the slider.
```
func main() {
	a := app.New()
	w = a.NewWindow("Hello")
	w.Resize(fyne.NewSize(960, 540))

	text := widget.NewTextGrid()
	scroll := container.NewScroll(text)
	label := widget.NewLabel("aaaaaaaaaaaaaaaaaa")
	content := container.NewVSplit(
		label,
		scroll,
	)
	content.SetOffset(0.01)
	w.SetContent(content)
	go func() {
		<-time.After(time.Millisecond * 100)
		s1 := text.Text()
		for strings.Count(s1, "\n") > 100 {
			pos := strings.Index(s1, "\n")
			s1 = s1[pos+1:]
		}
		s1 = strings.TrimPrefix(s1+"\n"+"55555555555555555555", "\n")

		text.SetText(s1)
		scroll.Refresh()
	}()
	w.ShowAndRun()
}
```
If the `scroll` become `label`, dragging the slider is too smooth on windows 10.

**Comment by andydotxyz** [comment, marked as answer]
This is due to text grid rather than the Split. There are some optimisations needed there.

**Reply by Bluebugs** [reply]
The part I don't understand is where is the buffering of all those input happening.  I would be thinking we have a lot of resize that let to redraw, but shouldn't they just end up reading the last input resize and jump over which shouldn't give that input buffering effect and large amount of redraw.

**Reply by andydotxyz** [reply]
There is no concept of "last input resize" as we do live resize - so every intermediate is reflected.
What we could do with adding is some realisation that a new value is being computed so skip old ones. But this is clearly a non-trivial problem.
